exercises/Loops-basic-index-R.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: exercise
+topic: Loops
+title: Basic Index
+language: R
+---
+
+The following code prints the first five whole numbers one line at a time:
+
+```
+for (i in 1:4){
+  print(i)
+}
+```
+
+1. Modify the code to print five orders of magnitude (*from one to ten
+   thousand*) one line at a time.
```

Comment:
I really like the general idea here, but I didn't understand what you were asking for with these "orders of magnitude" problems until I looked at the answer. The other issue I see here is that lots of bio folks won't easily think of the math required to understand how to do this (I speak from experience having removed problems in the last that assumed basic understanding of logarithms).

Maybe start with something simple "Three times each of the numbers from 1 to 10"? I like number 4. It might also be useful to do something that would get them to add at least one more line inside the loop. One other thing that might be useful to include is having them change the name of the variable so that they understand that `i` isn't special.